pkg/api/api_test.go

```diff
@@ -281,7 +281,7 @@ func newTestServer(t *testing.T, o testServerOptions) (*http.Client, *websocket.
 func request(t *testing.T, client *http.Client, method, resource string, body io.Reader, responseCode int) *http.Response {
 	t.Helper()
 
-	req, err := http.NewRequest(method, resource, body)
+	req, err := http.NewRequestWithContext(context.Background(), method, resource, body)
```

Comment:
The context should be used in the future, use `context.TODO()` instead to express this intention. The same is below.